### PR TITLE
[FIX] Marketplace tiered pricing plan wording

### PR DESCRIPTION
**Pull request by geekgonecrazy**
image
image

### CODE REVIEW

File: app/apps/client/admin/helpers.js [hunk 1]
```diff
@@ -363,11 +363,12 @@ export const appStatusSpanProps = ({
 
 export const formatPrice = (price) => `\$${ Number.parseFloat(price).toFixed(2) }`;
 
-export const formatPricingPlan = ({ strategy, price, tiers }) => {
+export const formatPricingPlan = ({ strategy, price, tiers = [] }) => {
 	const { perUnit = false } = (Array.isArray(tiers) && tiers.find((tier) => tier.price === price)) || {};
 
 	const pricingPlanTranslationString = [
 		'Apps_Marketplace_pricingPlan',
+		tiers.length > 0 && 'startingAt',
```
Comment (sampaiodiego) [inline app/apps/client/admin/helpers.js:371]: just to be safe, should it be tested here if `tiers` is an array just like on like 367?

Comment (geekgonecrazy) [reply app/apps/client/admin/helpers.js:371]: Probably but will it look ugly?

Comment (geekgonecrazy) [reply app/apps/client/admin/helpers.js:371]:
```suggestion
		Array.isArray(tiers) && tiers.length > 0 && 'startingAt',
```

Comment (sampaiodiego) [reply app/apps/client/admin/helpers.js:371]: the latter seems prettier =)

Comment (geekgonecrazy) [reply app/apps/client/admin/helpers.js]: committed then :)

**Pull request merged**